cli/loginCommands.js

```diff
@@ -93,7 +95,21 @@ const loginCommands = {
                 return;
             }
         }
-        await this.loginWithUserAndPassword();
+        if (this.isNotAlreadyLogged()) await this.loginWithUserAndPassword();
+    },
+
+    isNotAlreadyLogged() {
```

Comment:
Maybe it would be easier to understand if the function returns true if the user is already logged in ? If yes, I propose the function to be named : isUserAlreadyLoggedIn()